staging/src/k8s.io/apiserver/Godeps/Godeps.json

```diff
@@ -1657,6 +1657,10 @@
 		{
 			"ImportPath": "k8s.io/client-go/util/cert",
 			"Rev": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/util/flowcontrol",
```

Comment:
This is auto generated run update-staging-godep.sh,  as `"ImportPath": "k8s.io/client-go/util/cert",`  `"ImportPath": "k8s.io/client-go/util/flowcontrol",` is also imported before, and here occur twice. Sound a little weird.

Reply:
What do you mean with "occur twice"? Do we get "k8s.io/client-go/util/flowcontrol"  twice?